src/main/java/net/countercraft/movecraft/combat/features/tracking/KillCredCommand.java

```diff
@@ -0,0 +1,39 @@
+package net.countercraft.movecraft.combat.features.tracking;
+
+import net.countercraft.movecraft.combat.localisation.I18nSupport;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandExecutor;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+import org.jetbrains.annotations.NotNull;
+
+public class KillCredCommand implements CommandExecutor {
+    @Override
+    public boolean onCommand(@NotNull CommandSender sender, @NotNull Command command, @NotNull String label, @NotNull String... args) {
+        if (sender instanceof Player) {
+            Player p = (Player) sender;
+            if (p.hasPermission("movecraft.combat.hidekillcred")) {
```

Comment:
A better method to reduce code complexity is to send the no permission message and return false here, leaving the following code indented one less line.  See [here](https://github.com/TylerS1066/Movecraft-Combat/blob/613c7f70e23aa47226cbd6ff7a09f3a4720a4d80/src/main/java/net/countercraft/movecraft/combat/features/combat/CombatRelease.java#L126) for one example.

Reply:
Another note: It is usually good practice to `return true` if you have handled a command, this signals to Bukkit that you have accepted the command and handled it, even if that is to deny permission.  False is reserved for cases where you are given the wrong command.  Ref: [CommandExecutor](https://hub.spigotmc.org/javadocs/bukkit/org/bukkit/command/CommandExecutor.html)

Reply:
Fixed